pytim/__init__.py

```diff
@@ -306,9 +306,14 @@ def assign_universe(self, universe, radii_dict=None, warnings=False):
             self.interface.warnings = warnings
             if radii_dict is not None:
                 self.interface.radii_dict.update(radii_dict)
+            if isinstance(universe, MDAnalysis.core.universe.Universe):
+                self.interface.universe = universe
+            elif isinstance(universe, MDAnalysis.core.groups.AtomGroup):
```

Comment:
We can assume the user wants to pass the AtomGroup as the one for the surface analysis, so here one should also handle `itim_group`